bbs/load_board_summary.go

```diff
@@ -0,0 +1,31 @@
+package bbs
+
+import (
+	"fmt"
+
+	"github.com/Ptt-official-app/go-pttbbs/ptt"
+)
+
+func LoadBoardSummary(uuserID UUserID, bboardID BBoardID) (summary *BoardSummary, err error) {
+	bid, bbb, err := bboardID.ToRaw()
+	fmt.Println(string(bbb[:]))
```

Comment:
hi @cftang0827 ～

You can use

bid, _, err := bboardID.ToRaw()
to avoid dummy variables.

Reply:
Oh, it's my debug messages, I forgot to remove them. Will clear all debug messages in the code